resources/idlharness.js

```diff
@@ -1414,9 +1365,9 @@ IdlInterface.prototype.get_inheritance_stack = function() {
  * for inclusion in the default toJSON operation for easy
  * comparison with actual value
  */
-IdlInterface.prototype.default_to_json_operation = function(callback) {
+IdlInterface.prototype.default_to_json_operation = function() {
     var map = new Map(), isDefault = false;
-    this.traverse_inherited_and_consequential_interfaces(function(I) {
+    this.get_inheritance_stack().reverse().forEach(function(I) {
```

Comment:
This made me a little nervous because `reverse` operates in-place, and `get_inheritance_stack` doesn't signal very clearly that it returns a fresh array. It seems all is fine, though.

Followup: of the three callers of `get_inheritance_stack`, one only calls it for the side effects and the two others iterate backward. Maybe change `get_inheritance_stack` to return the array reversed already?

Reply:
I'll send a follow-up to rename it to `get_reverse_inheritance_stack`.

Reply:
https://github.com/web-platform-tests/wpt/pull/28688